source/macos/AdaptiveCards/AdaptiveCards/Renderers/InputNumberRenderer.swift

```diff
@@ -177,7 +180,11 @@ extension ACRNumericTextField: InputHandlingViewProtocol {
     }
     
     var value: String {
-        return String(Int(inputValue))
+        if Float(inputValue).truncatingRemainder(dividingBy: 1) == 0 {
+            return String(Int(inputValue))
```

Comment:
Isn't this same as Int(inputValue) ??

Reply:
Int(12.3) returns 12, so I was doing this to make sure the number is a perfect integer